http/restful-ws/src/main/java/io/cloudevents/http/restful/ws/CloudEventsProvider.java

```diff
@@ -99,6 +99,7 @@ public void writeTo(CloudEvent event, Class<?> type, Type genericType, Annotatio
             .map(a -> ((StructuredEncoding) a).value())
             .findFirst();
 
+        httpHeaders.remove("Content-Type");
```

Comment:
I'm not sure how this relates?

Reply:
I am going to completely remove those changes, to explain: the header is set to text/html by default, but we don't won't that there.